extensions/positron-assistant/src/participants.ts

```diff
@@ -288,6 +294,12 @@ abstract class PositronAssistantParticipant implements IPositronAssistantPartici
 						// TODO: Remove this restriction when the tool is supported in R https://github.com/posit-dev/positron/issues/8343
 						// The logic above with TOOL_TAG_REQUIRES_ACTIVE_SESSION will handle checking for active sessions once this is removed.
 						return activeSessions.has('python');
```

Comment:
Should we check `hasVariables` here as well?
```suggestion
						return activeSessions.has('python') && hasVariables;
```

Reply:
yes, we should do this

<img width="277" height="227" alt="image" src="https://github.com/user-attachments/assets/6bf8126b-232f-48d1-a88f-e8da5250c3b4" />